cmd/edgectl/aes_install.go

```diff
@@ -83,6 +86,15 @@ func aesInstall(cmd *cobra.Command, args []string) error {
 	kcontext, _ := cmd.Flags().GetString("context")
 	i := NewInstaller(verbose)
 
+	// If Scout is disabled (environment variable set to non-null), inform the user.
+	// TODO: should be i.scout.Disabled() when Alex's PR #2496 is merged.
+	if os.Getenv("SCOUT_DISABLE") != "" {
+		i.show.Printf(color.Info.Sprintf(phoneHomeDisabled))
+	}
```

Comment:
Now that #2496 has landed on the release branch, you can update this.

Reply:
will do

Reply:
done